### DISCUSSION TITLE
Is it expected that after marking a file or folder (with `m`) - they're gone if I close and reopen Vim?

### DISCUSSION THREAD

**Discussion by dracan** [nvim-tree/nvim-tree.lua#1636] [Q&A] [answered]
Wasn't sure whether to raise a bug, or if this is by design?

**Comment by alex-courtis** [comment, marked as answer]
Bookmarks are not durable across sessions.

We are considering adding that functionality: #1413

**Reply by dracan** [reply]
Great, thanks :)
